website/docs/index.html.markdown

```diff
@@ -185,6 +187,12 @@ The `features` block supports the following:
 
 ---
 
+The `api_management` block supports the following:
+
+* `purge_soft_delete_on_destroy` - (Optional) Should the `azurerm_api_management` resources be permanently deleted (e.g. purged) when destroyed? Defaults to `false`.
```

Comment:
We should also place a note in [azurerm_api_management](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/api_management) resource referring to this feature flag.